crates/wash-lib/src/cli/scale.rs

```diff
@@ -76,11 +89,62 @@ pub async fn handle_scale_component(cmd: ScaleComponentCommand) -> Result<Comman
         format!("{} max concurrent instances", cmd.max_instances)
     };
 
-    Ok(CommandOutput::from_key_and_text(
-        "result",
+    // If --skip-wait was specified,immediately return the result
+    if cmd.skip_wait {
+        return Ok(CommandOutput::from_key_and_text(
+            "result",
+            format!(
+                "Request to scale component {} to {scale_msg} has been accepted",
```

Comment:
Just double checking, did we add an additional sentence here like "this may take a few seconds to complete" for the other operations?

Reply:
Nope -- I don't see one on `start` or `stop` at the very least!